Modules/_elementtree.c

```diff
@@ -2059,6 +2073,20 @@ static PySequenceMethods element_as_sequence = {
     0,
 };
 
+static PyNumberMethods element_as_number = {
+    0,                                  /* nb_add */
+    0,                                  /* nb_subtract */
+    0,                                  /* nb_multiply */
+    0,                                  /* nb_remainder */
+    0,                                  /* nb_divmod */
+    0,                                  /* nb_power */
+    0,                                  /* nb_negative */
+    0,                                  /* nb_positive */
+    0,                                  /* nb_absolute */
+    element_bool,                       /* nb_bool */
```

Comment:
You can use a designated initializer (`.nb_bool = element_bool`) so you don't have to write out all the 0s.